webrtc/RTCRtpTransceiver-stop.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCRtpTransceiver.prototype.stop</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+async function negotiate(offerer, answerer) {
+  const offer = await offerer.createOffer();
+  await Promise.all([
+    offerer.setLocalDescription(offer),
+    answerer.setRemoteDescription(offer)
+  ]);
+  const answer = await answerer.createAnswer();
+  await Promise.all([
+    answerer.setLocalDescription(answer),
+    offerer.setRemoteDescription(answer)
+  ]);
+}
+
+promise_test(async ()=> {
+    const pc1 = new RTCPeerConnection();
+
+    pc1.addTransceiver("audio", { direction: "sendonly" });
+    pc1.addTransceiver("video");
+    pc1.getTransceivers()[0].stop();
+
+    const offer = await pc1.createOffer();
+    assert_false(offer.sdp.includes("m=audio"), "offer should not contain an audio m section");
+}, "An offer should be generated even if there is no valid transceiver");
+
+promise_test(async ()=> {
+    const pc1 = new RTCPeerConnection();
+    const pc2 = new RTCPeerConnection();
+
+    pc1.addTransceiver("audio");
+
+    await negotiate(pc1, pc2);
+
+    pc1.getTransceivers()[0].direction = "sendonly";
+    pc1.getTransceivers()[0].stop();
+
+    const offer = await pc1.createOffer();
+    assert_true(offer.sdp.includes("a=sendonly"), "m section should be sendonly");
+}, "A stopped sendonly transceiver should generate an inactive m section in the offer");
```

Comment:
This is testing that a stopped transceiver is present. That seems odd; you should end up with no m=audio section in the SDP.

Reply:
I see now - this test does negotiation, so it's already generated the section.
This test is not testing for port=0, which is the definition of stopped, though.

Reply:
Added an assert for port=0